### PR TITLE
Text update for GitHub and author list changes

### PR DESCRIPTION
**Pull request by ehsteve**
This PR makes each sentence one line to help with git version control. It also implements the author list discussion which occurred during the paper meeting of 8-May-2019.

### CODE REVIEW

File: sections/fido.tex [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+The result of a query can be inspected and edited before retrieval. 
+\Fido makes use of asynchronous download streams significantly improving download speeds. 
 
-A \Fido search can include multiple instruments, and can query all available data providers.  For example, a single search request can be issued that will query both the LYRA and RHESSI providers individually.  Data downloads via the \code{Fido.fetch} method can be split up into multiple parallel streams, improving download speeds.  \Fido also handles failed data downloads: the output from \code{Fido.fetch} flags failed downloads, and passing that output back in to \code{Fido.fetch}\ allows \Fido to attempt to download those files again.
+%Data downloads via the \code{Fido.fetch} method can be split up into multiple parallel streams, improving download speeds.  \Fido also handles failed data downloads: the output from \code{Fido.fetch} flags failed downloads, and passing that output back in to \code{Fido.fetch}\ allows \Fido to attempt to download those files again.
```
Comment (nabobalis) [inline sections/fido.tex:19]: This meant to be commented out?

**Comment by mbobra** [comment]
Does it make sense to add a pdf of the current version of the paper? It might be easier to read if we want to solicit comments.

File: sections/infrastructure.tex [hunk 2]
```diff
@@ -1,30 +1,61 @@
...
+
+\begin{itemize}
+\item \href{https://azure.microsoft.com/en-gb/services/devops/pipelines/}{Microsoft's Azure pipelines}: Runs the entire test suite on each operating system (Windows, Mac, Linux).
+\item \href{https://circleci.com}{circleci}: Runs the library's documentation, a special 32bit build and the figure tests.
+\item \href{https://travis-ci.org}{Travis CI}: Runs the entire test suite on a daily cadence.
+\item \href{https://codecov.io/}{Codecov}: Provides code coverage metrics.
+\end{itemize}
 
-\subsection{Package Template}
-Provided for affiliated packages
```
Comment (nabobalis) [inline sections/infrastructure.tex:30]: Where does this go now?

**Comment by ehsteve** [comment]
@mbobra I'd rather not commit a binary file (pdf) to the repo. We should have generated. Can someone make that happen?

**Comment by nabobalis** [comment]
You mean to generate one and host it somewhere? Or just this version?

File: sections/fido.tex (hunk 1 above)
Comment (ehsteve) [reply sections/fido.tex:19]: I'm not sure that it makes sense to compare the diffs. You should look at the latest overleaf to see if I did anything wrong.

File: sections/infrastructure.tex (hunk 2 above)
Comment (ehsteve) [reply sections/infrastructure.tex:30]: We will just add a sentence about it in the affiliated packages section.

**Comment by ehsteve** [comment]
@nabobalis I mean to generate based on the latest commit. I've seen it done somewhere.

**Comment by ehsteve** [comment]
They did it on the Astropy paper
https://github.com/astropy/astropy-v2.0-paper/blob/master/build-paper-travis.sh

**Comment by nabobalis** [comment]
AH. copy the script and set up travis should be "in theory" straightforward.

**Comment by Cadair** [comment]
We should do it on circle and we can configure Giles to preview the PDF on PRs like it does for the sunpy docs

File: sections/fido.tex (hunk 1 above)
Comment (wafels) [reply sections/fido.tex:19]: Yes, it is meant to be commented out.

**Pull request merged**